src/VolleyManagement.Backend/VolleyManagement.Services/TournamentService.cs

```diff
@@ -482,6 +504,17 @@ private void Archive(Tournament tournament)
             _tournamentRepository.Update(tournament);
         }
 
+        /// <summary>
+        /// Activate tournament.
+        /// </summary>
+        /// <param name="tournament">Tournament to archive.</param>
+        private void Activate(Tournament tournament)
```

Comment:
Why do you need this method in the first place?

Reply:
Another level of abstraction, I suppose. Like with equals. Move this behaviour in public method?

Reply:
yes, inline it please